lambdas/criteria/evaluate.py

```diff
@@ -8,6 +8,11 @@
 
 chat_client = OpenAI()
 
+COMMON_HEADERS = {
```

Comment:
Let's create a generate_response method similar to authentication and put it in there